standard/unsafe-code.md

```diff
@@ -884,7 +884,7 @@ fixed_size_buffer_declarator
     ;
 ```
 
-A fixed-size buffer declaration may include a set of attributes ([§21](attributes.md#21-attributes)), a `new` modifier ([§14.3.5](classes.md#1435-the-new-modifier)), a valid combination of the four access modifiers ([§14.3.6](classes.md#1436-access-modifiers)) and an `unsafe` modifier ([§22.2](unsafe-code.md#222-unsafe-contexts)). The attributes and modifiers apply to all of the members declared by the fixed-size buffer declaration. It is an error for the same modifier to appear multiple times in a fixed-size buffer declaration.
+A fixed-size buffer declaration may include a set of attributes ([§21](attributes.md#21-attributes)), a `new` modifier ([§14.3.5](classes.md#1435-the-new-modifier)), any one of the permitted kinds of declared accessibility ([§14.3.6](classes.md#1436-access-modifiers)) and an `unsafe` modifier ([§22.2](unsafe-code.md#222-unsafe-contexts)). The attributes and modifiers apply to all of the members declared by the fixed-size buffer declaration. It is an error for the same modifier to appear multiple times in a fixed-size buffer declaration.
```

Comment:
Earlier we have

> Fixed-size buffers are only permitted in struct declarations

and

> Since inheritance isn’t supported for structs, the declared accessibility of a struct member cannot be `protected`, `private protected`, or `protected internal`.

And yet now

> A fixed-size buffer declaration may include ... any one of the permitted kinds of declared accessibility ...

I suspect this is incorrect.  See the ANTLR for `fixed_size_buffer_modifier` to see what declared accessibilities are permitted, and also see [HERE](https://sharplab.io/#v2:EYLgtghglgdgPgVxgZwgMwKYAJkBcBOCAxrlgBpYDeAsAFBYNYAO+UAbhLtiwPZckYAJljRQAHkKyxSAOQDaARgAMAXQDcdAL5A=) demonstrating that this text incorrect.

Reply:
(All of the rest of the changes look good)

Reply:
Hmm... I'm confused as to why `protected` is in `fixed_size_buffer_modifier` then. I suspect that's a bug a few lines higher... will look at both that and the text today. Thanks for spotting this.

Reply:
Following @gafter I think possible changes are:

> … any one of the permitted kinds of declared accessibilities for struct members (§16.4.3)

and to remove the `| 'protected'` alternative from the `fixed_size_buffer_modifier` grammar rule

Reply:
I've modified this slightly, to differentiate between the modifiers and the declared accessibilities they represent.

(I've added the change as a new commit.)